project_4/index.html

```diff
@@ -0,0 +1,384 @@
+                    your passion. We
+                    believe every designer should be informed about sustainability, and the cost of our actions if we
+                    continue designing in the way we design now. Sustainanbility should be on every designer's mind
+                    before they begin a project whether you are a graphic designer, fashion designer, or a product
+                    designer; it's all
+                    relevant. It is also important to hear from a variety of practicers and perspectives from every
+                    field, because believe it or not, a lot of fields of design are interconnected; it is important
+                    to learn about their practices moving forward in the age of our ever changing climate and
+                    economy.</p>
+            </div>
+
+            <div class="quotes">
+                <h2 data-aos="fade-right">What Are People Saying?</h2>
+                <blockquote class="quote">"From top to bottom the speakers and topics were carefully curated. The
+                    execution was flawlessly organized. I made so many connections. Professional development has never
+                    been so fun."</blockquote>
+                <!-- https://revolveconference.com/ -->
+                <p> - Douglas Davis, Creative Strategist - </p>
+                <button type="button" class="button-reset button-next">Next</button>
+            </div>
+        </section>
+
+        <!-- SPEAKERS -->
+        <h2 class="section-header">Speakers</h2>
+        <section id="speakers" class="speakers">
+
+            <ul class="speaker-info" data-aos="fade-up" data-aos-anchor-placement="top-center">
+                <a href="https://www.linkedin.com/in/maxinebedat" title="Link to Maxine Bedat's LinkedIn page">
+                    <li><img src="images/Maxine_Bedat.jpg" alt=""></li>
+                </a>
+                <!-- left the alt text blank for the headshot images because they do not provide new or helpful information--the name and info are the most important-->
+                <li class="speaker-name">Maxine Bedat</li>
+                <li class="speaker-title">Executive Director at New Standard Institute</li>
+                <li class="speaker-paragraph">Founder and director of the New Standard Institute. She
+                    co-founded the fashion brand, Zady, focusing on sustainable
+                    practices with the best materials, low environmental footprints,
+                    fair labor, and style. Zady was named of the world’s “Most Innovative
+                    Companies” in retail.</li>
+                <!-- image and information from https://truecostmovie.com/maxine-bedat-interview/ -->
+            </ul>
+
+            <ul class="speaker-info" data-aos="fade-up" data-aos-anchor-placement="top-center">
+                <a href="https://www.krisdedecker.org/" title="Link to Kris De Decker's Personal Website">
```

Comment:
I like the `title` even more now that the link actually goes to different places -- this is a helpful way to let folks know what they're getting into before they click.